algorithms/FindersKeepers.js

```diff
@@ -0,0 +1,6 @@
+// Create a function that looks through an array (first argument) and returns the first element in the array that passes a truth test (second argument).
```

Comment:
As a code quality practice, I prefer to keep lines of code less than 80 characters, to make it easier to read on smaller screen sizes.  If this comment is important, I suggest breaking it into two lines to facilitate ease of reading.